### ISSUE TITLE
Hover (via click) was broken in iPhone 7 between 4.0.1 and 4.1.1 versions

### ISSUE THREAD

**Issue by silveoj** [valor-software/ng2-bootstrap#5642]
Bug description:

We updated ```ngx-boostrap``` package from 3.* to 5.* version.

But we found issue on iPhone 7:
we have hover state in desktop. iPhone can emulate it (without our code) if you click by element which has hover.
But it stopped to work (real iPhone and via BrowserStack (iPhone 10, 12)). After investigation we detected it was broken between 4.0.1 and 4.1.1 versions.

We suppose this commit https://github.com/valor-software/ngx-bootstrap/commit/4d49218a993fd60b67d3a43f73f2489263fd0a35
We use only datapicker, modal, pagination, collapse, tooltip, dropdown.
This page (route) doesn't contain any ngx-bootstrap components.

Workaround: use ```onclick=""``` (empty) for DOM element.

Plunker/StackBlitz that reproduces the issue:
I can't reproduce this issue in empty StackBlitz project
https://angular-ngx-bootstrap-hover-iphone7-issue.stackblitz.io

Versions:
- ngx-bootstrap: between 4.0.1 (work) and 4.1.1 (doesn't work)
- Angular: 7 or 8
- Bootstrap: any 4.*

Build system: Angular CLI, webpack, starter seed:

**Expected behavior**

Hover (via click) in iPhone 7 shows content.

**Comment by dmitry-zhemchugov** [comment]
Hello @silveoj !  I'm sorry but I can't reproduce this issue. Could you please record a video with reproducing it?

**Comment by silveoj** [comment]
I prepared videos but I think they are useless.
We can remove empty ```onclick``` from DIVs and add ```onclick=""``` into body. It works also.

I suppose it must not work in iPhone 7 before this commit https://github.com/valor-software/ngx-bootstrap/commit/4d49218a993fd60b67d3a43f73f2489263fd0a35
But ```ngx-bootstrap``` "saved" us because code before this commit registered global document click event (like ```onclick``` in body). So it worked and we can't notice it.

In this commit ```ngx-bootstrap``` removed global event and we ran into popular iPhone issue/thing.

I'm sure it's not ```ngx-bootstrap``` issue.
Thanks for quick reply.
